// [[Reason: TS resolves invalid types to any]] ////

//// [tests/cases/compiler/parseImportAttributesError.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,10 +1,10 @@
 
 
 //// [/.src/out/index.d.ts]
 export type LocalInterface = import("pkg", { with: {} });
-export declare const a: any;
-export declare const b: any;
+export declare const a: import("pkg", { with: {} });
+export declare const b: import("pkg", { with: {} });
 
 /// [Errors] ////
 
 error TS2468: Cannot find global value 'Promise'.
